src/types.js

```diff
@@ -263,3 +263,27 @@ export type PmConversationData = {|
   recipients: string,
   unread: number,
 |};
+
+
+/**
+ * URL return type of parse-url
+ */
+export type UrlType={
```

Comment:
The right place for this is not in `src/` but in `flow-types/npm/url-parse_v1.3.x.js` – where it can already be found.